frontend/src/components/document/Comments/SidebarComment.jsx

```diff
@@ -1,91 +1,104 @@
+import HoverableIcon from "@/components/common/HoverableIcon";
+import MenuSVG from "@/assets/meatballs_menu.svg";
+import PurpleMenuSVG from "@/assets/meatballs_menu_purple.svg";
 
-import ProfileIcon from "@/assets/pfp.svg"
+import ProfileIcon from "@/assets/pfp.svg";
 
 import { useRecoilValue } from "recoil";
 import { activeCommentThreadIDAtom } from "@/util/CommentRedactionState";
-import { useState, useEffect } from "react" 
+import { useState, useEffect } from "react";
 import { getCommentThreadsOnTextNode } from "@/util/editorCommentUtils";
 
-import { removeSelectedMark } from "@/util/editor_utils"
+import { removeSelectedMark } from "@/util/editor_utils";
 
 import { useSlate } from "slate-react";
-import { deleteCommentFromDocument } from "@/util/localDocStore"
+import { deleteCommentFromDocument } from "@/util/localDocStore";
 import { getMarkForCommentThreadID } from "@/util/editorCommentUtils";
 
-import { format } from "date-fns";
+import { format, parseISO } from 'date-fns';
 
-function SidebarComment ({id, comment, docId}) {
+function SidebarComment({ id, comment, docId }) {
+  const [isFocus, setIsFocus] = useState(false);
+  const activeCommentThreadID = useRecoilValue(activeCommentThreadIDAtom);
+  const editor = useSlate();
 
-    const [isFocus, setIsFocus] = useState(false)
-    const activeCommentThreadID = useRecoilValue(activeCommentThreadIDAtom);
-    const editor = useSlate();
+  //convert to comment format
+  const formatCreationTime = (creationTimeISO) => {
+    const date = parseISO(creationTimeISO);
+    return format(date, 'MMM dd, yyyy hh:mm aaaa');
+  };
 
-    useEffect (() => {
-        if (activeCommentThreadID) {
-            const activeCommentMarkSet = getCommentThreadsOnTextNode(activeCommentThreadID);   
-            setIsFocus(activeCommentMarkSet.has(id));
-        } else {
-            setIsFocus(false)
-        }
-    }, [activeCommentThreadID, id]);
-
-    const handleResolveComment = () => {
-        removeSelectedMark(editor, getMarkForCommentThreadID(id));
-        deleteCommentFromDocument(docId, id);
+  useEffect(() => {
+    if (activeCommentThreadID) {
+      const activeCommentMarkSet = getCommentThreadsOnTextNode(
+        activeCommentThreadID
+      );
+      setIsFocus(activeCommentMarkSet.has(id));
+    } else {
+      setIsFocus(false);
     }
+  }, [activeCommentThreadID, id]);
 
-    //TODO: implement menu
-    const handleClickMenu = () => {
-        //console.log("menu")
-    }
+  const handleResolveComment = () => {
+    removeSelectedMark(editor, getMarkForCommentThreadID(id));
+    deleteCommentFromDocument(docId, id);
+  };
 
-    //TODO: for now, you can only focus on comments when you click the corresponding comments
-    const handleOnClick = () => {
-        // setIsFocus(!isFocus)
-        // if (isFocus) {
-        //     setActiveCommentThreadID(id);
-        //     console.log(activeCommentThreadID)
-        // }
-    }
+  //TODO: implement menu
```

Comment:
what's the menu for?

Reply:
edit/delete popup? definitely a stretch goal so maybe i will just delete